### PR TITLE
Label

### PR DESCRIPTION
**Pull request by nicolad**

### CODE REVIEW

**Comment by nicolad** [comment]
image

File: packages/core/package.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
     "@emotion/react": "^11.10.4",
     "@emotion/styled": "^11.10.4",
     "@shared/theme": "*",
+    "css": "link:@types/styled-system/css",
```
Comment (nicolad) [inline packages/core/package.json:29]: could u please remove this, forgot to remove

cc @marinanicolai

Comment (marinanicolai) [reply packages/core/package.json:29]: @nicolad 
Screenshot from 2023-02-10 06-23-37

I did, and I got this.
Should I revert back to previous version?

Comment (nicolad) [reply packages/core/package.json:29]: Yes, revert for now

**Pull request merged**